src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -1,33 +1,31 @@
-﻿<?xml version="1.0"?>
+<?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.Orleans.CounterControl</id>
```

Comment:
This is based on the assumption that InstallUtil will work in all environments as well as the CounterControl.  That it has no unexpected dependencies and/or side effects.  That when errors occur they will be clear enough to users.  These are all probably true, but we've very large and critical services running on Orleans and 1.3 already includes a fair amount of risk.  Continuing to support this assembly until the new approach has been vetted in production environments is simply a risk mitigation strategy that costs us little but can save us and our customers hours (or days) should something go wrong.